packages/backend/src/submission/mongoRepository.ts

```diff
@@ -62,8 +57,19 @@ async function findInvitationById(collection: Collection, id: string) {
 }
 
 async function addInvitation(collection: Collection, invitation: Invitation) {
-  const invitationData = { ...invitation }
-  const result = await collection.insertOne(invitationData)
+  const result = await collection.insertOne({ ...invitation })
   logger.info(`Invitation inserted to the database`, result)
   return invitation
 }
+
+async function updateInvitation(
```

Comment:
How/when is this function called? 🤔

Reply:
It seems we don't use it right? We can remove it if not.

Reply:
It would be called when approving an submission. i.e. if the invitation associated with the submission has no entityId, then the invitation would be updated with the new entityId to ensure future submissions created with that invitationUrl correspond to the same entity.

Reply:
So should we remove or keep it in place for the future?

Reply:
Remove :)